superset/assets/src/components/FilterableTable/FilterableTable.jsx

```diff
@@ -29,14 +30,52 @@ import {
   Table,
 } from 'react-virtualized';
 import { getTextDimension } from '@superset-ui/dimension';
+import { t } from '@superset-ui/translation';
+
+import Button from '../Button';
+import CopyToClipboard from '../CopyToClipboard';
+import ModalTrigger from '../ModalTrigger';
 import TooltipWrapper from '../TooltipWrapper';
 
 function getTextWidth(text, font = '12px Roboto') {
   return getTextDimension({ text, style: { font } }).width;
 }
 
+function getJsonStringifiedObject(data) {
+  try {
```

Comment:
In which scenario do you expect it to fail?
You can also use `../../../src/utils/safeStringify` if the crash cases are circular json.

Reply:
It’ll fail if we don’t pass in valid json. Like “foo” or “{“. Let me know if there’s an easier way to determine if a string is valid json and resolves to an object/array

Reply:
It’ll fail if we don’t pass in valid json. Like “foo” or “{“. Let me know if there’s an easier way to determine if a string is valid json and resolves to an object/array